kadi/commands/validate.py

```diff
@@ -678,6 +678,30 @@ class ValidateHETG(ValidateGrating):
     plot_attrs = PlotAttrs(title="HETG", ylabel="HETG")
 
 
+class ValidateSunPosMon(ValidateStateCode):
+    state_name = "sun_pos_mon"
+    msids = ["aopssupm"]
+    plot_attrs = PlotAttrs(title="Sun position monitor", ylabel="Sun position monitor")
+    min_violation_duration = 300
```

Comment:
This 300 second value is somewhat empirical from testing. It seems there is variation between the actual eclipse exit (determined by the OBC based on batteries etc) and the predicted which is in commanding. With a 300 second value most of the cases were within the limit.